src/pollution_model/run_pollution_model.py

```diff
@@ -0,0 +1,454 @@
+    second_stage_copper = b[6] * rainfall_ph ** b[7]
+
+    initial_zinc_concentration = (c[0] * rainfall_ph + c[1]) * (c[2] * antecedent_dry_days ** c[3]) * (
+            c[4] * average_rain_intensity ** c[5])
+    second_stage_zinc = c[6] * rainfall_ph + c[7]
+
+    # Define Z as per experimental data
+    z = 0.75
+
+    # Define K, the wash off coefficient. TODO: find out what the k parameter should be
+    k = 1
+
+    # Initialise total copper and zinc loads as a guaranteed common factor
+    total_copper_load = initial_copper_concentration * surface_area * (1 / k)
+    total_zinc_load = initial_zinc_concentration * surface_area * (1 / k)
+
+    # Calculate total metal loads, where the method depends on if Z is less than event_duration
+    if event_duration <= z:
+        factor = (1 - math.exp(k * average_rain_intensity * event_duration))
+        total_zinc_load *= factor
+        total_copper_load *= factor
+    else:
+        factor = (1 - math.exp(k * average_rain_intensity * z))
+        bias_factor = average_rain_intensity * (event_duration - z)
+        total_zinc_load = total_zinc_load * factor + second_stage_zinc * surface_area * bias_factor
+        total_copper_load *= total_copper_load * factor + second_stage_copper * surface_area * bias_factor
+
+    return total_copper_load, total_zinc_load
+
+
+def total_metal_load_road_carpark(tss_surface):
+    """
+    Calculate the total metal load for car parks and roads from their total suspended solids.
+
+    Parameters
+    ----------
+    tss_surface: float
+        total suspended solids of this surface
+
+    Returns
+    -------
+    float, float
+       Returns the total copper and zinc loads for this surface
+    """
+    # Define constants
+    proportionality_constant_cu = 0.441
+    proportionality_constant_zn = 1.96
+    # Return total copper load, total zinc load
+    return tss_surface * proportionality_constant_cu, tss_surface * proportionality_constant_zn
+
+
+def dissolved_metal_load(total_copper_load, total_zinc_load, surface_type):
+    """
+        Calculate the dissolved metal load for all surfaces from their total suspended solids.
+
+        Parameters
+        ----------
+        total_copper_load: float
+            total copper load for the surface
+        total_zinc_load: float
+            total zinc load for the surface
+        surface_type: int
+            The type of surface that we are calculating this for
+
+        Returns
+        -------
+        float, float
+           Returns the dissolved copper and zinc loads for this surface
+        """
+    # Declare constants
+    f = 1
+    g = 1
+    # Set constant values based on surface type
+    match surface_type:
+        case "Cr":
+            f = 0.46
+            g = 0.67
+        case "Cu":
+            f = 0.77
+            g = 0.72
+        case "Gv":
+            f = 0.28
+            g = 0.43
+        case "Rd" | "CrP":
+            f = 0.28
+            g = 0.43

```

Comment:
These should be in the form `case SurfaceType.CONCRETE_ROOF:` to make it independent of the implementation details